assets/src/js/pages/dashboard.js

```diff
@@ -27,22 +28,47 @@ class Dashboard extends Component {
     this.state = {
       dateRange: [],
       groupBy: 'day',
-      isPublic: document.cookie.indexOf('auth') < 0,
+      isPublic: props.autherized(),
```

Comment:
Can you change this property name to `authorized`?

Reply:
Whoops, I can't spell :-1:

Reply:
Fixed in a1b0321.